Automattic/wp-calypso
# A4A Dev Sites: Add development badge to A4A licenses page.

Closes https://github.com/Automattic/dotcom-forge/issues/9038

## Proposed Changes

* Add development badge to A4A Licenses page.
* Round the corners of badges on the A4A Licenses page.

## Why are these changes being made?


* See pdtkmj-2Te-p2#comment-5571

## Testing Instructions



1. Apply D161261-code on your sandbox.
2. Go to https://agencies.automattic.com/purchases/licenses.
3. You should see development badge on dev sites.

![Screenshot 2024-09-17 at 22 33 20](https://github.com/user-attachments/assets/866d333c-f07a-4f66-8cde-07203492c621)


![Markup on 2024-09-13 at 12:34:08](https://github.com/user-attachments/assets/8948663a-17b1-4f46-b694-d2d6dc5a4d4d)

## Pre-merge Checklist
